corehq/apps/locations/tests/test_bulk_management.py

```diff
@@ -967,3 +974,173 @@ def test_case_sensitivity(self):
         self.assertLocationTypesMatch(FLAT_LOCATION_TYPES)
         self.assertLocationsMatch(self.as_pairs(lower_case))
         self.assertCouchSync()
+
+    def test_partial_addition(self):
+        # new locations can be added without having to specify all of old ones
+        self.bulk_update_locations(
+            FLAT_LOCATION_TYPES,
+            self.basic_tree
+        )
+
+        addition = [
+            ('State 3', 's3', 'state', '', '', False) + extra_stub_args,
```

Comment:
nit: I think in the future you could use a `namedtuple` here with default properties, so that you don't need to always add the `extra_stub_args` to every single one of these.